seeders/usersSeeder.ts

```diff
@@ -0,0 +1,91 @@
+import { PrismaClient } from "@prisma/client";
+import { faker } from "@faker-js/faker";
+import { random } from "./randomFunction";
+
+const prisma = new PrismaClient();
+
+const seedUsers = async (prisma: PrismaClient): Promise<void> => {
+  const users = [];
+  const documentTypes = ["CC", "CE", "PP"];
+  const userRoles = ["user", "restaurantAdmin", "appAdmin"];
```

Comment:
I would recommend an object instead of array, for future mantainance:
```
const userRoles = {
  user: "user",
  "restaurantAdmin": "restaurantAdmin",
  "appAdmin": "appAdmin",
}
```